CustomMetadata/SamlMetadataCustomConfiguration Example.json

```diff
@@ -0,0 +1,380 @@
+{
+    "nemlogin2": {
```

Comment:
nemlogin2 does not support aes gcm or those required attributes. You can remove nemlogin2 from this example.

Reply:
remove